android/app/src/main/res/values/colors.xml

```diff
@@ -105,7 +105,7 @@
   <color name="button_accent_disabled">@color/black_12</color>
   <color name="button_accent_disabled_night">#FF505050</color>
   <color name="button_accent_text">#FFFFFFFF</color>
-  <color name="button_accent_text_night">#FF000000</color>
+  <color name="button_accent_text_night">@color/white_primary</color>
```

Comment:
Why this color was changed?

Reply:
Before the color was black in night mode for button text which was not looking good
I took reference of ios version of OM there it was white in night mode so i changed the whole button text color to white in night

I can revert it to black also

Reply:
It would be great to see screenshots from old and new versions to better understand the change.

Reply:
Nevermind, I see them at the bottom. Thanks!